### ISSUE TITLE
Add Deno

### ISSUE THREAD

**Issue by tmokmss** [maxday/lambda-perf#1364]
It appears Deno is faster than Node.js or Bun. https://deno.com/blog/aws-lambda-coldstart-benchmarks

It'd be great if this benchmark add Deno runtime.

**Comment by gitblit** [comment]
I agree. I came here looking to see if there was an open request to confirm the claim. Would be great to see this datapoint amongst the other JS runtimes and at varying lambda size - which is something not considered by blog post.

**Comment by soundstep** [comment]
https://github.com/maxday/lambda-perf/pull/1467

**Comment by soundstep** [comment]
Hi @maxday (sorry to ping directly), any chance someone can have a look at that one?
https://github.com/maxday/lambda-perf/pull/1467
Cheers.

**Comment by markhaslam** [comment]
If helpful for implementing this I just noticed that Deno archived the deno-lambda repo (https://github.com/denoland/deno-lambda) just today, and then point to their official documentation on how to deploy Deno to AWS Lambda here: https://docs.deno.com/runtime/tutorials/aws_lambda/

**Comment by soundstep** [comment]
Cheers, I closed that PR and check what that all means.

**Comment by vicary** [comment]
@soundstep Out of curiosity I tried to build a myself following the official docs and was greeted with the error `The image manifest or layer media type for the source image...` on all combinations of build arch and runtime arch, would love to see how you do it!

**Comment by soundstep** [comment]
FYI, new PR: https://github.com/maxday/lambda-perf/pull/1492

**Comment by maxday** [comment]
Thanks a lot! Will take a look at it now
